bobpay/public/pay/index.html

```diff
@@ -41,15 +41,26 @@
     function pay() {
       if(!paymentRequestClient) return;
 
-      var paymentAppResponse = {
-        methodName: "https://emerald-eon.appspot.com/bobpay",
-        details: {
-          id: "123456"
-        }
-      };
-
-      paymentRequestClient.postMessage(paymentAppResponse);
-      window.close();
+      var authn = new WebAuthnTransaction();
```

Comment:
Here we should feature-detect for web authentication functionality to avoid breaking the demo for those who don't have it turned on:

```javascript
if (window.WebAuthnTransacttion) {
  var authn = new WebAuthnTransaction();
  // Web Authentication flow.
} else {
  var paymentAppResponse = {
  // The existing flow without Web Authentication.
}
```